ui/src/app/app-routing.module.ts

```diff
@@ -34,6 +35,11 @@ const routes: Routes = [
     component: ProjectsComponent,
     canActivate: [CapabilitiesActivator]
   },
+  {
+    path: 'dashboard',
+    component: DashboardComponent,
+    canActivate: [CapabilitiesActivator]
```

Comment:
One thing we'll need to handle in the CapabilitiesActivator: the dsub dashboard page will require a project (probably via query parameter). If it's not there, we'd want to redirect to the projects page (similar to how we're redirecting from "jobs" -> "projects" if there is no projectId in the query).

Feel free to handle this in this PR, or else just put a TODO in the CapabilitiesActivator